Okta.psm1

```diff
@@ -1,4 +1,4 @@
-﻿$ExecutionContext.SessionState.Module.OnRemove = {
```

Comment:
Ok, i'm staring at this line and i cannot for the life of me tell what is different... is this just a git glitch?

Reply:
no, it had a funny character at the beginning (Unicode BOM?) so i deleted it. 

don't worry about this line...